diff --git a/src/platform/windows/display_vram.cpp b/src/platform/windows/display_vram.cpp
index 558c3bf..af4a69f 100644
--- a/src/platform/windows/display_vram.cpp
+++ b/src/platform/windows/display_vram.cpp
@@ -2094,7 +2094,7 @@ namespace platf::dxgi {
       }
     } else if (adapter_desc.VendorId == 0x8086) {  // Intel
       // If it's not a QSV encoder, it's not compatible with an Intel GPU
-      if (!boost::algorithm::ends_with(name, "_qsv")) {
+      if (!boost::algorithm::ends_with(name, "_qsv") && !boost::algorithm::ends_with(name, "_mf")) {
         return false;
       }
       if (config.chromaSamplingType == 1) {
